test/unit/storage/patch/SparqlUpdatePatchHandler.test.ts

```diff
@@ -148,6 +148,25 @@ describe('A SparqlUpdatePatchHandler', (): void => {
     ])).toBe(true);
   });
 
+  it('handles composite updates.', async(): Promise<void> => {
+    await handler.handle({ identifier: { path: 'path' },
+      patch: { algebra: translate(
+        'INSERT DATA { <http://test.com/s1> <http://test.com/p1> <http://test.com/o1>. ' +
+        '<http://test.com/s2> <http://test.com/p2> <http://test.com/o2> };' +
+        'DELETE WHERE { <http://test.com/s1> <http://test.com/p1> <http://test.com/o1>.' +
+        '<http://test.com/startS1> <http://test.com/startP1> <http://test.com/startO1> }',
```

Comment:
Check!

Can we just also have a unit test for the (common) `DELETE DATA/INSERT DATA` pattern?
And perhaps an integration test, too?
Example at https://github.com/solid/community-server/issues/485

Reply:
I'll add one, but just for the record, the following single query is also supported:
```
DELETE { <https://css.verborgh.org/home/lights#3> <urn:tmp:philips:hue#hue> "1338"^^<http://www.w3.org/2001/XMLSchema#integer>. }
INSERT { <https://css.verborgh.org/home/lights#3> <urn:tmp:philips:hue#hue> "1337"^^<http://www.w3.org/2001/XMLSchema#integer>. }
WHERE {}
```